app/furniture/layouts/marketplace.html.erb

```diff
@@ -1,22 +1,28 @@
 <%- content_for :content do %>
-  <%= turbo_frame_tag(marketplace, data: { turbo_action: :advance }) do %>
-    <%- if policy(marketplace).edit? %>
-      <%= render Marketplace::ManagementComponent.new(marketplace: marketplace) do %>
-        <%= yield  %>
+  <%- if room.hero_image&.upload.present? %>
+    <%= image_tag room.hero_image&.upload&.variant(resize_to_fill: Media::FULL_WIDTH_SHORT), class: "w-full" %>
+  <%- end %>
```

Comment:
I think this makes sense here?

Reply:
In practice I didn't notice any double rendering. With the way the templates are "stacked" currently this seems to work fine. I spot checked this everywhere I could.

I'm not even that strong on having this image here. But it seemed nicer than a stark order summary while I was going through the checkout flow. Although the more I sit with it I think presenting the shopper the summary without visual embellishment might be a more focused experience. So I'll take it out.